std_msgs/Header header
common/Uint8TimeStamped cdcu_ehb_workmode
common/Uint8TimeStamped cdcu_ehb_brkmode
common/FloatTimeStamped cdcu_ehb_brkpresur
common/Uint8TimeStamped cdcu_ehb_errlevel